samples/metrics_monitor/CMakeLists.txt

```diff
@@ -22,36 +22,24 @@ list( APPEND LIBS cttmetrics)
 
 make_executable( metrics_monitor none )
 
-# test_monitor
+#test_monitor
 
 pkg_check_modules(PKG_LIBDRM    libdrm)
 pkg_check_modules(PKG_PCIACCESS pciaccess)
 
 if( PKG_LIBDRM_FOUND AND
-    PKG_PCIACCESS_FOUND AND
-    DEFINED GTEST_FOUND)
+    PKG_PCIACCESS_FOUND )
 
-    set( sources "" )
+    file( GLOB_RECURSE test_srcs "${CMAKE_CURRENT_SOURCE_DIR}/test/*.cpp" )
 
-    list ( APPEND incldirs
-        ${GTEST_INCLUDE}/gtest
-        ${GTEST_INCLUDE}/gtest/internal
-        ${CMAKE_CURRENT_SOURCE_DIR}/test )
+    add_executable( test_monitor ${test_srcs})
+    target_include_directories( test_monitor PUBLIC ./include ${PKG_LIBDRM_INCLUDE_DIRS})
+    target_link_libraries( test_monitor pthread rt gtest cttmetrics ${PKG_LIBDRM_LIBRARIES})
 
-    foreach( dir ${incldirs} )
-      include_directories( ${dir} )
-    endforeach( )
+    set_target_properties(test_monitor PROPERTIES
+    RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BIN_DIR}/${CMAKE_BUILD_TYPE})
 
-    file( GLOB_RECURSE srcs "${CMAKE_CURRENT_SOURCE_DIR}/test/*.cpp" )
-    list( APPEND sources ${srcs})
-    list( APPEND LIBS
-      pthread
-      rt
-      gtest )
-
-    set(DEPENDENCIES libdrm)
-
-    make_executable( test_monitor none )
+    install(TARGETS test_monitor RUNTIME DESTINATION ${MFX_APPS_DIR}/tests)
```

Comment:
This line is a partial fix for #752. Can you, please, extract this in a separate commit and separate PR? And please, add install commands for the other targets from #752.